pkg/transformers/json/transformer.go

```diff
@@ -140,3 +171,26 @@ func flatten(prefix string, m, m1 map[string]interface{}) (map[string]interface{
 	}
 	return m, nil
 }
+
+func transformTimestamp(payload []byte) (int64, error) {
```

Comment:
No need to pass `payload []byte`, since it is already unmarshalled [here](https://github.com/mainflux/mainflux/pull/1514/files#diff-0c018aa1ba72aec369a1c61fb548029880389d3e8a802e641a2ea1e6e3ad86c5L56). Only pass `interface{}` and check if its `map[string]interface{}` and parse. This will prevent unnecessary `json.Unmarshal` calls that can be quite expensive.